config/admins.txt

```diff
@@ -138,3 +138,4 @@ crazypilot = Game Master
 vivalas = Game Master
 ninjanomnom = Game Master
 cdey = Game Master
+Wilchen = Host
```

Comment:
we use SQL for admins you tard

Reply:
adding yourself to the admins txt is a long held tradition